cpp/src/reductions/reduction_operators.cuh

```diff
@@ -16,10 +16,10 @@
 
 #pragma once
 
-#include <utilities/cudf_utils.h>  //for CUDA_HOST_DEVICE_CALLABLE
+#include <utilities/legacy/cudf_utils.h>  //for CUDA_HOST_DEVICE_CALLABLE
```

Comment:
Non-legacy files should not depend on legacy files. `CUDA_HOST_DEVICE_CALLABLE` is defined in `types.hpp`.